Formula/apache-arrow.rb

```diff
@@ -1,8 +1,8 @@
 class ApacheArrow < Formula
   desc "Columnar in-memory analytics layer designed to accelerate big data"
   homepage "https://arrow.apache.org/"
-  url "https://www.apache.org/dyn/closer.cgi?path=arrow/arrow-0.12.0/apache-arrow-0.12.0.tar.gz"
-  sha256 "34dae7e4dde9274e9a52610683e78a80f3ca312258ad9e9f2c0973cf44247a98"
+  url "https://archive.apache.org/dist/arrow/arrow-0.12.1/apache-arrow-0.12.1.tar.gz"
```

Comment:
We should not use `*.apache.org` for download.
https://github.com/Homebrew/homebrew-core/pull/36768#discussion_r254875296

Reply:
The dynamic mirror thing of Apache, together with their tendency to remove older files, makes it hard for us. Let's note that most of our downloads come from Homebrew bottles anyway, which are hosted on bintray.